<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExecWindow</class>
 <widget class="QMainWindow" name="ExecWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>토끼와 거북이</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>카메라 on / off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_status">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>520</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>종료하기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_webcam">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>641</width>
      <height>421</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_setting">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>설정</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
